pkgs/servers/scylladb/default.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv,
+  fetchgit,
+  python3Packages,
+  python3,
+  pkgconfig,
+  gcc8,
+  boost,
+  git,
+  systemd,
+  gnutls,
+  cmake,
+  makeWrapper,
+  ninja,
+  ragel,
+  hwloc,
+  jsoncpp,
+  libantlr3cpp,
+  antlr3,
+  numactl,
+  protobuf,
+  cryptopp,
+  libxfs,
+  libyamlcpp,
+  libsystemtap,
+  lksctp-tools,
+  lz4,
+  libxml2,
+  zlib,
+  libpciaccess,
+  snappy,
+  libtool
+}:
+let
+  thriftNixpkgs = builtins.fetchGit {
+    name = "thrift-0.10";
+    url = https://github.com/nixos/nixpkgs/;
+    rev = "fbaa12bad90e3c8726a5427fdd981dc60c7f08ff";
+  };
+  thriftPkgs = import thriftNixpkgs {};
+in
+stdenv.mkDerivation rec {
+  name = "scylladb-${version}";
+  version = "3.0.5";
+
+  src = fetchgit {
+    url = "https://github.com/scylladb/scylla.git";
+    rev = "403f66ecad6bc773712c69c4a80ebd172eb48b13";
+    sha256 = "14mg0kzpkrxvwqyiy19ndy4rsc7s5gnv2gwd3xdwm1lx1ln8ywsi";
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./seastar-configure-script-paths.patch ];
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [
+   python3Packages.pyparsing
+   python3
+   gcc8
+   boost
+   git
+   systemd
+   gnutls
+   cmake
+   makeWrapper
+   ninja
```

Comment:
Tools that are only needed at build time like ninja, makeWrapper, etc. should go into `nativeBuildInputs`

Reply:
Will try to filter them out. Any straightforward way to do this automatically?